README.md

```diff
@@ -20,6 +20,11 @@ The following is the request flow (seen in blue in the architecture diagram abov
 1. The Redis Source component sends cloud events to our Consumer service
 1. The consumer component reads the cloud event and synchronously makes the service call to the Knative Service.
 
+## Prerequisites
+- A kubernetes environment, recommended version and sizing [here](https://knative.dev/docs/install/knative-with-operators/#prerequisites)
+- Installation of Istio before installing Knative Serving and Eventing
```

Comment:
If you follow the knative installation instructions (via .yaml) Istio is actually installed as a part of those instructions. For the operators, it's listed as a pre-req for that installation method. I don't think you should include it here as it's referenced from those other 2 places :) 

Additionally, this component will not always be Istio only.